lib/Request.js

```diff
@@ -155,10 +154,10 @@ class Request {
    * Promise).
    */
 
-  each(func, isAsync) {
+  each(func, isAsync, concurrency) {
```

Comment:
actually I think I see what you're saying...forEach sets the concurrency to `1`, so chaining off the same context leaves the concurrency that way?